Digipost/SHCOAuthViewController.m

```diff
@@ -171,6 +178,21 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
     return YES;
 }
 
+- (void) remove1PasswordButtonIfNotNormalLoginScope {
+    if (self.scope != kOauth2ScopeFull) {
+        self.navigationItem.rightBarButtonItem = nil;
+    }
+}
+
+- (IBAction)fillUsing1Password:(id)sender {
```

Comment:
Fine ryddinger 👍 . Kunne f.eks. si at vi legger oss opp mot https://github.com/NYTimes/objective-c-style-guide

For min del trenger vi ikke en linter; er egentlig mer fan av at vi er enige om hvordan vi gjør det og at vi drar det i retning av kode _vi_ synes ser bra ut.